tests/13.switch.out

```diff
@@ -0,0 +1,26 @@
+[translated]
+module main
+
+[export: 'a']
+const (
+	a = 0
+)
+
+fn main() {
+	match a {
+		0 /* case comp body kind=CallExpr is_enum=true */ {
+			C.printf(c'bar\n')
+		}
+		else {
+			C.printf(c'foo\n')
+		}
+	}
+	match a {
```

Comment:
Why is match here twice?  That looks like a bug.

Reply:
There are 2 switchs in the code.

Reply:
Oh, I see.  There are 2 in your test case, not in the original report.